src/xcode/ENA/ENA/Source/Scenes/ContactDiary/Store/__tests__/MockDiaryStore.swift

```diff
@@ -4,7 +4,8 @@
 
 import Foundation
 import OpenCombine
-@testable import ENA
+// CJE: Move back to ENATests target
+// @testable import ENA
```

Comment:
Still TODO?

Reply:
ToDo once the store is fully implemented in another branch